test/hitl/test_hitl_examples.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import multiprocessing
+import runpy
+import sys
+from os import path
+
+import pytest
+
+
+def run_main(*args):
+    sys.argv = list(args)
+    target = args[0]
+    if path.isfile(target):
+        sys.path.insert(0, path.dirname(target))
+    runpy.run_path(target, run_name="__main__")
+
+
+def run_main_as_subprocess(args):
+    context = multiprocessing.get_context("spawn")
+    process = context.Process(target=run_main, args=args)
+    process.start()
+    process.join()
+    assert process.exitcode == 0
+
+
+@pytest.mark.parametrize(
+    "args",
+    [
+        (
+            "examples/hitl/basic_viewer/basic_viewer.py",
+            "--config-dir",
+            "test/config/habitat_hitl",
+            "+experiment=smoke_test",
```

Comment:
This uses the [Hydra experiment workflow](https://hydra.cc/docs/patterns/configuring_experiments/). I'm intending to have more than one in the future.